### DISCUSSION TITLE
Which 🪐️ in our solar system is the closest to the Kuiper belt?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#889] [Galaxy Trivia] [answered]
**Bonus:** Which dwarf planets are officially recognized in the Kuiper belt?

**Comment by peterbaccaro** [comment, marked as answer]
The planet in our solar system that is closest to the Kuiper Belt is **Neptune**. The Kuiper Belt is a region of space beyond the orbit of Neptune that contains a large number of small celestial bodies, including dwarf planets such as Pluto and Eris. While Neptune itself is not located within the Kuiper Belt, it is the closest major planet to this region, and its gravitational influence helps shape the structure and dynamics of the Kuiper Belt objects.

There are **five** officially recognized dwarf planets in the Kuiper Belt:

- Pluto: Pluto was discovered in 1930 and was considered the ninth planet in our solar system until its reclassification as a dwarf planet in 2006. It is the largest known object in the Kuiper Belt and has a complex system of moons, including its largest moon, Charon.

- Eris: Eris was discovered in 2005 and played a significant role in the reclassification of Pluto. It is slightly smaller than Pluto but is more massive. Eris is known for its highly elongated orbit and is one of the largest dwarf planets in the solar system.

- Haumea: Haumea was discovered in 2004 and is notable for its elongated, football-like shape. It rotates rapidly, completing one rotation in less than four hours. Haumea also has two known moons, Hi'iaka and Namaka.

- Makemake: Makemake was discovered in 2005 and is the third-largest known dwarf planet in the Kuiper Belt. It is slightly smaller than Haumea and is known for its reddish surface. Makemake has one known moon called S/2015 (136472) 1.

- Gonggong (formerly 2007 OR10): Gonggong is the largest unnamed dwarf planet in the solar system. It was discovered in 2007 and has a highly eccentric orbit. Gonggong is currently awaiting an official name from the International Astronomical Union (IAU).
